.github/workflows/copy-completed-notebooks.yml

```
@@ -0,0 +1,117 @@
+          path: training-modules
+
+      - name: Checkout this repository
+        uses: actions/checkout@v3
+        with:
+          path: main
+
+      - name: Configure git
+        run: |
+          # use global flag since we have multiple repos here
+          git config --global --add safe.directory "${GITHUB_WORKSPACE}"
+          git config --global user.email "actions@github.com"
+          git config --global user.name "GitHub Actions"
+
+      - name: Get HTML completed notebooks from training-modules
+        shell: bash
+        run: |
+          # The completed notebooks to copy will depend on which workshop module this is.
+          # For each module, the directory of notebooks to copy are:
+          #  Intro R: intro_to_R_tidyverse/
+          #  Intro scRNA-seq: intro_to_R_tidyverse/ and scRNA-seq/
+          #  Advanced scRNA-seq: advanced-scRNA-seq/
+
+          # the url for for this tag of the training-modules repo where files will be downloaded from
+          base_url=https://raw.githubusercontent.com/AlexsLemonade/training-modules/${{ github.event.inputs.training-modules-tag }}
+
+          ########### Get paths for files to copy, for the given module ############
+
+          # We have to cd into training-modules to ensure correct paths come out of `ls`
+          cd training-modules
+
+          # First, intro-to-R-tidyverse notebooks for the intro R workshop
+          if [[ "${{ github.event.inputs.training-module }}" == "Introduction to R and Tidyverse" ]]; then
+            notebook_files=`ls intro-to-R-tidyverse/*html`
+          fi
+
+          # Second, intro-to-R-tidyverse notebooks and scRNA-seq notebooks for the intro scRNA-seq workshop
+          if [[ "${{ github.event.inputs.training-module }}" == "Introduction to single-cell RNA-seq" ]]; then
+            notebook_files=`ls intro-to-R-tidyverse/*html scRNA-seq/*html`
+          fi
+
+          # Third, scRNA-seq-advanced notebooks for the advanced scRNA-seq workshop
+          if [[ "${{ github.event.inputs.training-module }}" == "Advanced single-cell RNA-seq" ]]; then
+            notebook_files=`ls scRNA-seq-advanced/*html`
+          fi
+
+          # Now, we have to navigate back to the completed-notebooks directory
+          cd ../main/completed-notebooks
+
+          # curl completed notebook htmls into main/completed-notebooks
+          for path in ${notebook_files}
+          do
+            # test that the path is to an .html file, error if not
+            # note that at least one notebook is only .html, NOT .nb.html, so we'll just use .html
+            [[ $path != *.html ]] && echo "'$path' is not an .html file." && exit 1
+            echo ${base_url}/${path}
+            curl --fail -sO ${base_url}/${path}
+          done
+
+      - name: Create PR with rendered notebooks
+        uses: peter-evans/create-pull-request@v4
+        id: cpr
+        with:
+          path: main # must be a RELATIVE path to _this_ repo
```

Just continuing my suggestion here
```suggestion
          path: website # must be a RELATIVE path to _this_ repo
```